README.md

```diff
@@ -96,3 +96,29 @@ class ExampleModel(Model):
     description     = columns.Text(required=False)
 sync_table(ExampleModel)
 ```
+
+## Connect to Google Sheets
+
+You can use load_sheet_to_athena() function to load Google sheet data into an Athena table.
```

Comment:
Add snippets of 
1. how to Connect
1. how to instantiate Sheet Util
1. how to read a dataframe using SheetUtil